site/zenodo_rdm/serializers/bibtex.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+"""Zenodo bibtex serializer."""
+
+from flask_resources import BaseListSchema, MarshmallowSerializer
+from flask_resources.serializers import SimpleSerializer
+from invenio_rdm_records.resources.serializers import BibtexSerializer
+from invenio_rdm_records.resources.serializers.bibtex.schema import BibTexSchema
+from marshmallow import fields, missing
+
+
+class ZenodoBibtexSchema(BibTexSchema):
+    """Zenodo bibtex schema."""
+
+    swhid = fields.Method("get_swhid")
+
+    def get_swhid(self, obj):
+        """Get swhid."""
+        return obj.get("swh", {}).get("swhid") or missing
```

Comment:
For `bibtex`, this is not enough because the schema imports a `formatter` that I also had to change.